### PR TITLE
removed notstarted status

### PR DESCRIPTION
**Pull request by santhoshb-msft**
Removed unused NotStarted status.

### CODE REVIEW

**Comment by code4clouds** [comment]
@santhoshb-msft  This may need a rebase..

File: src/SaaS.SDK.Services/Services/SubscriptionService.cs [hunk 1]
```diff
@@ -195,7 +195,7 @@ public List<SubscriptionResult> GetPartnerSubscriptions(string partnerEmailAddre
         /// <returns> Subscription Status EnumExtension.</returns>
         public SubscriptionStatusEnumExtension GetSubscriptionStatus(string subscriptionStatus)
         {
-            var status = SubscriptionStatusEnumExtension.NotStarted;
+            SubscriptionStatusEnumExtension status;
```
Comment (code4clouds) [inline src/SaaS.SDK.Services/Services/SubscriptionService.cs:198]: This is missing the default value to return...

Try.Enum is usually inside an IF statement.  IF True returns the out, if False then dev needs to set the default value.   Ref the language spec: https://docs.microsoft.com/en-us/dotnet/api/system.enum.tryparse?view=net-6.0

**Review by code4clouds — CHANGES_REQUESTED** [review]
Need change... on the Enum.Try

File: src/SaaS.SDK.Services/Services/SubscriptionService.cs (hunk 1 above)
Comment (code4clouds) [reply src/SaaS.SDK.Services/Services/SubscriptionService.cs:198]: @santhoshb-msft ... any ETA on this change?

Comment (santhoshb-msft) [reply src/SaaS.SDK.Services/Services/SubscriptionService.cs]: Done! @code4clouds

**Review by code4clouds — APPROVED** [review]

**Pull request merged**